package.json

```diff
@@ -37,121 +37,92 @@
   },
   "private": true,
   "dependencies": {
-    "@angular-devkit/build-angular": "~12.2.18",
-    "@angular/animations": "~12.2.17",
-    "@angular/cdk": "^12.2.13",
-    "@angular/common": "~12.2.17",
-    "@angular/compiler": "~12.2.17",
-    "@angular/core": "~12.2.17",
-    "@angular/forms": "~12.2.17",
-    "@angular/platform-browser": "~12.2.17",
-    "@angular/platform-browser-dynamic": "~12.2.17",
-    "@angular/router": "~12.2.17",
+    "@angular-devkit/build-angular": "~14.2.13",
+    "@angular/animations": "~14.3.0",
+    "@angular/cdk": "^14.2.7",
+    "@angular/common": "~14.3.0",
+    "@angular/compiler": "~14.3.0",
+    "@angular/core": "~14.3.0",
+    "@angular/forms": "~14.3.0",
+    "@angular/platform-browser": "~14.3.0",
+    "@angular/platform-browser-dynamic": "~14.3.0",
+    "@angular/router": "~14.3.0",
     "@auth0/angular-jwt": "5.0.0",
-    "@awesome-cordova-plugins/core": "^5.1.0",
-    "@awesome-cordova-plugins/health": "^5.43.0",
-    "@awesome-cordova-plugins/in-app-browser": "^5.36.0",
-    "@awesome-cordova-plugins/keyboard": "^5.39.0",
-    "@ionic-native/android-permissions": "^5.26.0",
-    "@ionic-native/app-launcher": "^5.31.1",
-    "@ionic-native/app-version": "^5.28.0",
-    "@ionic-native/background-mode": "^5.32.0",
-    "@ionic-native/barcode-scanner": "^5.33.1",
-    "@ionic-native/core": "^5.32.0",
-    "@ionic-native/device": "^5.26.0",
-    "@ionic-native/dialogs": "^5.26.0",
-    "@ionic-native/file": "^5.32.1",
-    "@ionic-native/firebase": "^5.31.1",
-    "@ionic-native/firebase-x": "^5.29.0",
-    "@ionic-native/globalization": "^5.26.0",
-    "@ionic-native/insomnia": "^5.32.1",
-    "@ionic-native/keyboard": "^5.26.0",
-    "@ionic-native/local-notifications": "^5.28.0",
-    "@ionic-native/mobile-accessibility": "^5.28.0",
-    "@ionic-native/network": "^5.36.0",
-    "@ionic-native/splash-screen": "^5.26.0",
-    "@ionic-native/status-bar": "^5.26.0",
-    "@ionic-native/vibration": "^5.32.1",
-    "@ionic-native/web-intent": "^5.28.0",
-    "@ionic/angular": "^5.9.4",
+    "@capacitor-community/barcode-scanner": "^4.0.1",
```

Comment:
This is deprecated. Can we use [@capacitor-mlkit/barcode-scanning](https://github.com/capawesome-team/capacitor-mlkit/tree/main/packages/barcode-scanning) instead?

Reply:
Thanks I've updated this now.